pkg/policy/parser.go

```diff
@@ -69,7 +71,12 @@ func DecodePolicy(body hcl.Body, diags hcl.Diagnostics, basePath string) (*Polic
 			panic("unexpected block")
 		}
 	}
-	return nil, diags
```

Comment:
Isn't the `for _, block := range content.Blocks` a bit strange as you can't really have multiple `policies`?